client/src/js/services/NotifyService.js

```diff
@@ -31,17 +62,48 @@ function NotifyService($interval) {
    *
    * @todo extend for optiosn
    */
-  function success(message) {
+  function success(key, ttl) {
+    setNotification(key, ttl, formatOptions.success);
+  }
+
+  function danger(key, ttl) {
+    setNotification(key, ttl, formatOptions.danger);
+  }
+
+  function info(key, ttl) {
+    setNotification(key, ttl, formatOptions.info);
+  }
+
+  function warn(key, ttl) {
+    setNotification(key, ttl, formatOptions.warn);
+  }
+
+  /**
+   * This method is not simply a formatting alias - it accepts an error object
+   * and parses it to show relevent information in the notification.
+   * By default the time to live of an error notification is significantly longer.
+   */
+  function handleError(error) {
+    var ERROR_TTL = 60000;
```

Comment:
Potentially you might want to move this to the top of the service - it would allow much easier finding and configuration if we wish to give the option to configure it later.